src/osd/modules/font/font_windows.cpp

```diff
@@ -85,10 +85,9 @@ bool osd_font_windows::open(std::string const &font_path, std::string const &_na
 	logfont.lfPitchAndFamily = DEFAULT_PITCH | FF_DONTCARE;
 
 	// copy in the face name
-	TCHAR *face = tstring_from_utf8(name.c_str());
-	_tcsncpy(logfont.lfFaceName, face, sizeof(logfont.lfFaceName) / sizeof(TCHAR));
+	std::basic_string<TCHAR> face = tstring_from_utf8(name.c_str());
+	_tcsncpy(logfont.lfFaceName, face.c_str(), sizeof(logfont.lfFaceName) / sizeof(TCHAR));
```

Comment:
can the ARRAY_LENGTH template for getting number of elements in the array not be used here?